pygments/formatters/img.py

```diff
@@ -7,6 +7,7 @@
     :copyright: Copyright 2006-2024 by the Pygments team, see AUTHORS.
     :license: BSD, see LICENSE for details.
 """
+from packaging import version
```

Comment:
`packaging` isn't an stdlib module. It's often installed because pip needs it, but I think that, e.g., a vanilla Debian system doesn't have it. I'd just do a `hasattr` check here.

Reply:
f53e3b77cb1bfe931c1aec6cc55eb2888a3dcb78